static/app/gettingStartedDocs/javascript/react.tsx

```diff
@@ -0,0 +1,144 @@
+    });
+
+    const container = document.getElementById(“app”);
+    const root = createRoot(container);
+    root.render(<App />)
+    `,
+  },
+  {
+    type: StepType.VERIFY,
+    description: t(
+      "This snippet contains an intentional error and can be used as a test to make sure that everything's working as expected."
+    ),
+    code: `
+    return <button onClick={() => methodDoesNotExist()}>Break the world</button>;
+    `,
+  },
+];
+
+const nextSteps = [
+  {
+    id: 'source-maps',
+    name: t('Source Maps'),
+    description: t('Learn how to enable readable stack traces in your Sentry errors.'),
+    link: 'https://docs.sentry.io/platforms/javascript/guides/react/sourcemaps/',
+  },
+  {
+    id: 'react-features',
+    name: t('React Features'),
+    description: t('Learn about our first class integration with the React framework.'),
+    link: 'https://docs.sentry.io/platforms/javascript/guides/react/features/',
+  },
+  {
+    id: 'performance-monitoring',
+    name: t('Performance Monitoring'),
+    description: t(
+      'Track down transactions to connect the dots between 10-second page loads and poor-performing API calls or slow database queries.'
+    ),
+    link: 'https://docs.sentry.io/platforms/javascript/guides/react/performance/',
+  },
+  {
+    id: 'session-replay',
+    name: t('Session Replay'),
+    description: t(
+      'Get to the root cause of an error or latency issue faster by seeing all the technical details related to that issue in one visual replay on your web application.'
+    ),
+    link: 'https://docs.sentry.io/platforms/javascript/guides/react/session-replay/',
+  },
+];
+
+// Configuration End
+
+type Props = {
+  activeProductSelection: PRODUCT[];
+  dsn: string;
+};
+
+export default function GettingStartedWithReact({dsn, activeProductSelection}: Props) {
+  const integrations: string[] = [];
+  const otherConfigs: string[] = [];
+  let nextStepDocs = [...nextSteps];
+
+  if (activeProductSelection.includes(PRODUCT.PERFORMANCE_MONITORING)) {
+    integrations.push(performanceIntegration);
+    otherConfigs.push(performanceOtherConfig);
+    nextStepDocs = nextStepDocs.filter(step => step.id !== 'performance-monitoring');
+  }
+
+  if (activeProductSelection.includes(PRODUCT.SESSION_REPLAY)) {
+    integrations.push(replayIntegration);
+    otherConfigs.push(replayOtherConfig);
+    nextStepDocs = nextStepDocs.filter(step => step.id !== 'session-replay');
+  }
+
+  if (integrations.length > 0) {
+    integrations.unshift('integrations: [');
+    integrations.push('],');
+  }
+
+  const sentryInitContent: string[] = [`dsn: "${dsn}"`];
+
+  if (integrations.length > 0) {
+    sentryInitContent.push(integrations.join('\n'));
+  }
+
+  if (otherConfigs.length > 0) {
+    sentryInitContent.push(otherConfigs.join('\n'));
+  }
+
+  return (
+    <Layout
+      language={language}
+      steps={steps(sentryInitContent.join('\n'))}
+      nextSteps={nextStepDocs}
+    />
+  );
+}
```

Comment:
this configuration would have to be copied and pasted for other platforms as well. No engineers would not need to do anything here.